lib/combo-renderer.coffee

```diff
@@ -101,7 +102,7 @@ module.exports =
     @lastStreak = performance.now()
     @debouncedEndStreak()
 
-    n = n * (@level + 1) if n > 0
+    n = n * (@level + 1) if n > 0 and @conf['multiplier']
```

Comment:
> @lexcast  
     You need to check with this config on the `checkLevel` function as well to avoid triggering the multiplier 
     exclamations (those 2x, 3x).

Already fixed.